### PR TITLE
FEATURE: FET-294 updated babel

### PR DESCRIPTION
**Pull request by yanlipnican**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-app-components:
- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3432/ : SUCCESS in 21m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4252/ : SUCCESS in 1m 09s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3528/ : SUCCESS in 9m 34s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4376/ : SUCCESS in 15s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1880/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1223/ : UNSTABLE in 16m 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4377/ : SUCCESS in 6s

**Comment by yanlipnican** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-946.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3555/ : SUCCESS in 17m 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4409/ : SUCCESS in 11s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1239/ : FAILURE in 25m 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4410/ : SUCCESS in 13s

**Comment by yanlipnican** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-946.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1240/ : FAILURE in 26m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4411/ : SUCCESS in 9s

**Comment by yanlipnican** [comment]
extended test - examples

**Comment by yanlipnican** [comment]
extended test - examples

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by yanlipnican** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-946.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1241/ : FAILURE in 27m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4412/ : SUCCESS in 15s

**Comment by yanlipnican** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-946.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1243/ : FAILURE in 28m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4418/ : SUCCESS in 11s

**Comment by yanlipnican** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-946.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3573/ : SUCCESS in 6m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4434/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1251/ : SUCCESS in 28m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4435/ : SUCCESS in 19s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1253/ : FAILURE in 11m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4439/ : SUCCESS in 11s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1908/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1259/ : UNSTABLE in 12m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4447/ : SUCCESS in 16s

**Comment by yanlipnican** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-946.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3580/ : SUCCESS in 6m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4449/ : SUCCESS in 11s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1260/ : FAILURE in 28m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4450/ : SUCCESS in 8s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3589/ : SUCCESS in 7m 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4467/ : SUCCESS in 9s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1915/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1269/ : UNSTABLE in 12m 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4468/ : SUCCESS in 10s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3591/ : FAILURE in 4m 57s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4470/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3593/ : SUCCESS in 5m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4473/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1271/ : FAILURE in 1h 00m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4474/ : SUCCESS in 20s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1919/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1273/ : UNSTABLE in 12m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4476/ : SUCCESS in 17s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3595/ : SUCCESS in 5m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4478/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1274/ : FAILURE in 12m 34s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4479/ : SUCCESS in 10s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1922/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1276/ : UNSTABLE in 10m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4481/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3597/ : SUCCESS in 3m 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4483/ : SUCCESS in 8s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1925/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1278/ : SUCCESS in 9m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4485/ : SUCCESS in 11s

File: examples/webpack.config.js [hunk 1]
```diff
@@ -42,26 +43,28 @@ module.exports = async (env) => {
...
+            },
+            onProxyReq(proxyReq) {
+                proxyReq.setHeader('accept-encoding', 'identity')
             }
         },
         '/api': {
             target: 'http://localhost:3009',
             secure: false,
             onProxyReq: (req) => {
+                console.log('proxy', '/gdc', req.path); // eslint-disable-line no-console
```
Comment (no23reason) [inline examples/webpack.config.js]: Now we have two console logs on each call. Is this necessary?

File: .storybook/webpack.config.js [hunk 2]
```diff
@@ -39,7 +39,15 @@ module.exports = {
                     path.join(__dirname, '../stories'),
                     path.join(__dirname, '../src')
                 ],
-                loaders: ['awesome-typescript-loader?configFileName=./stories/tsconfig.json']
```
Comment (no23reason) [inline .storybook/webpack.config.js]: Why did you migrate to `ts-loader`? At least in my experience `awesome-typescript-loader` is usually quite faster.
We use `awesome-typescript-loader` in KD (storybooks) and it works just fine :)

File: examples/webpack.config.js [hunk 3]
```diff
@@ -42,26 +43,28 @@ module.exports = async (env) => {
 
     // see also production proxy at /examples/server/src/endpoints/proxy.js
     const proxy = {
-        '/gdc': {
-            target: backendUrl,
-            secure: false,
-            cookieDomainRewrite: '',
-            onProxyReq: (proxyReq) => {
-                console.log('proxy', '/gdc', proxyReq.path); // eslint-disable-line no-console
-                if (proxyReq.method === 'DELETE' && !proxyReq.getHeader('content-length')) {
```
Comment (martinmosko) [inline examples/webpack.config.js]: this check is not needed any more right ? :)

File: examples/webpack.config.js [hunk 4]
```diff
@@ -108,8 +111,7 @@ module.exports = async (env) => {
                 booleans: true,
                 unused: true,
                 if_return: true,
-                join_vars: true,
-                warnings: false
```
Comment (martinmosko) [inline examples/webpack.config.js]: so false is a default ? :)

File: examples/webpack.config.js [hunk 5]
```diff
@@ -78,7 +81,7 @@ module.exports = async (env) => {
     };
 
     const plugins = [
-        new CleanWebpackPlugin(['dist']),
```
Comment (martinmosko) [inline examples/webpack.config.js]: ['dist'] argument not needed anymore?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3599/ : SUCCESS in 7m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4486/ : SUCCESS in 20s

File: package.json [hunk 6]
```diff
@@ -13,18 +13,28 @@
     "LICENSE"
   ],
   "devDependencies": {
+    "@babel/core": "^7.4.4",
```
Comment (martinmosko) [inline package.json]: I'm not sure if we already agreed to use `^` on external dependencies... or something changed?

File: examples/webpack.config.js [hunk 7]
```diff
@@ -122,17 +124,8 @@ module.exports = async (env) => {
                 uglifyOptions,
                 parallel: true
             }),
-            new CompressionPlugin({
-                asset: '[file].gz',
```
Comment (martinmosko) [inline examples/webpack.config.js]: this configuration was moved ? or its default config and its not needed anymore?

File: examples/webpack.config.js [hunk 8]
```diff
@@ -42,26 +43,28 @@ module.exports = async (env) => {
...
-                if (proxyReq.method === 'DELETE' && !proxyReq.getHeader('content-length')) {
-                    // Only set content-length to zero if not already specified
-                    proxyReq.setHeader('content-length', '0');
-                }
-
-                proxyReq.setHeader('host', backendUrl.split('/')[2]); // White labeled resources are based on host header
-                proxyReq.setHeader('referer', backendUrl);
-                proxyReq.setHeader('origin', null);
+        "/gdc": {
+            "changeOrigin": true,
```
Comment (no23reason) [inline examples/webpack.config.js]: Formatting nitpick: the quotes around the key names are not necessary

File: stories/tsconfig.json [hunk 9]
```diff
@@ -3,6 +3,7 @@
     "extends": "../tsconfig.json",
     "include": ["../src/typings.d.ts", "./*.ts", "./*.tsx"],
     "exclude": ["src/**/*.spec.ts"],
+    "files": [""],
```
Comment (martinmosko) [inline stories/tsconfig.json]: what is the purpose? :)

Comment (no23reason) [inline stories/tsconfig.json]: Why is this needed?

File: examples/webpack.config.js (hunk 3 above)
Comment (yanlipnican) [reply examples/webpack.config.js]: I don't what it does, looks important. I moved it, it's not deleted.

File: examples/webpack.config.js (hunk 5 above)
Comment (yanlipnican) [reply examples/webpack.config.js]: Yes, it has different interface and it now cleans dist by default.

File: package.json [hunk 10]
```diff
@@ -119,9 +120,11 @@
     "lodash": "4.17.11",
     "nodelist-foreach-polyfill": "^1.2.0",
     "prop-types": "^15.6.0",
+    "react-async-script": "^1.0.1",
```
Comment (no23reason) [inline package.json]: Why was this added? We don't seem to use it anywhere. I know it was moved from `devDependencies`, but there is no reference to this package in the whole project.

File: package.json [hunk 11]
```diff
@@ -119,9 +120,11 @@
...
     "react-dom-factories": "^1.0.2",
     "react-intl": "^2.4.0",
     "react-measure": "2.3.0",
+    "react-transition-group": "2.2.1",
```
Comment (no23reason) [inline package.json]: Why was this added? We don't seem to use it anywhere.

File: examples/webpack.config.js (hunk 4 above)
Comment (yanlipnican) [reply examples/webpack.config.js]: This is not in the interface anymore.

File: examples/webpack.config.js (hunk 7 above)
Comment (yanlipnican) [reply examples/webpack.config.js]: Yeah this is default config.

File: stories/tsconfig.json (hunk 9 above)
Comment (yanlipnican) [reply stories/tsconfig.json]: Yes, this is some kind of bug, when `files` is empty it sometimes fails to build for some reason. This fixes problem

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3600/ : FAILURE in 3m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4487/ : SUCCESS in 9s

File: .storybook/webpack.config.js (hunk 2 above)
Comment (yanlipnican) [reply .storybook/webpack.config.js]: Yeah, i tested ts-loader with `transpileOnly: true` and it's same speed. And i think there are no other benefits using 3rd party loader.

File: examples/webpack.config.js (hunk 8 above)
Comment (yanlipnican) [reply examples/webpack.config.js]: Odd, that prettier did not catch this. I will fix that

File: examples/webpack.config.js (hunk 1 above)
Comment (yanlipnican) [reply examples/webpack.config.js]: I just moved this.

Comment (yanlipnican) [reply examples/webpack.config.js]: I think it's unnecessary too. But i did not want to change look and feel :D

File: package.json (hunk 11 above)
Comment (yanlipnican) [reply package.json]: I think it was needed peer dependency from something i will check that again. Thanks for catch

Comment (yanlipnican) [reply package.json]: It's peer dependency for `recharts` which is used as example of custom visualization

File: stories/tsconfig.json (hunk 9 above)
Comment (yanlipnican) [reply stories/tsconfig.json]: Well, i can't really explain why this works. But it stopped random build fails. When error message was something like `Files is empty blah blah` i think it's storybook quirk

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: package.json (hunk 10 above)
Comment (yanlipnican) [reply package.json]: It's also peer dependency

Comment (yanlipnican) [reply package.json]: I think it's recaptcha package

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3602/ : FAILURE in 3m 28s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4489/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3603/ : SUCCESS in 8m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4490/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3604/ : SUCCESS in 6m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4491/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1926/ : ABORTED in 1h 00m 02s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3605/ : SUCCESS in 4m 07s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/979/ : SUCCESS in 39s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1082/ : SUCCESS in 26m 36s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-react-components-storybook-zuul-docker : SKIPPED
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/980/ : FAILURE in 22s
- gooddata-react-components-examples-zuul-docker : SKIPPED

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1933/ : SUCCESS in 13m 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3612/ : SUCCESS in 7m 28s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/981/ : SUCCESS in 35s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1086/ : SUCCESS in 23m 46s
